### ISSUE TITLE
Can't get Radial to launch with Gw2Launcher

### ISSUE THREAD

**Issue by SOL4RIS** [Friendly0Fire/GW2Radial#266]
I'm unable to get GW2Radial to launch when I use Gw2Launcher to launch the game.  It works fine if I run the game straight from the Gw2-64.exe file, but with Launcher, nothing I've thought to try will get Radial to load.

My setup:
 - GW2 installed in the default Program Files directory
 - GW2 Add On Manager used to install GW2 Radial (and nothing else)
 - GW2 Launcher to run the game for multiple accounts
 - Blish HUD, set to launch on my main account only via Launcher

**Expected behavior**
Obviously, I expect GW2Radial to launch along with my game, but I've been unsuccessful thus far.  It did used to work when I was on Windows 10, but I recently installed Windows 11 and all GW2 related apps along with that clean install, and since then, I've been unable to get Radial to play nicely with Launcher.  I'm unsure if I've selected any Launcher settings that are breaking Radial with the newer version I installed.  I have tried both using GW2 Add On Manager to install Radial, as well as the manual installation instructions from the FAQs.  When going the manual route, I get an error about DirectX 9 not being installed (which it is), and the game won't launch at all.

**Screenshots**
Launch options for main account:
![2021-12-14 23_50_36-](https://user-images.githubusercontent.com/79487317/146137818-ff0571ca-5ea1-4ecb-bac2-f2a9dda91133.png)

Launch options in Settings:
![2021-12-14 23_52_59-](https://user-images.githubusercontent.com/79487317/146137946-3b925446-7d18-46fd-9c8b-67bf48efee2a.png)

**System configuration:**
 - OS: Windows 11
 - GPU: GTX 1060
 - Chainloading theoretically handled by GW2 Add On Manager

I'm happy to provide any additional information or screenshots needed to help troubleshoot the issue.  Thanks in advance!

**Comment by Friendly0Fire** [comment]
I personally use GW2Launcher and have no such issues, so this is a bit strange to me... I genuinely don't see what's wrong. The only suggestion I have would be (unfortunately) to try completely clearing GW2Launcher's configuration and the special folders it creates on the off chance some wonky setting remains there.

**Comment by SOL4RIS** [comment]
Thanks for the response.  I went ahead and updated Radial with the update manager (which oddly enough made it not work until I removed the gw2radial folder in `C:\Users\USER\Documents\Guild Wars 2\addons`).  Removed and reinstalled launcher and as of this moment, everything appears to be working again. I'll probably play a little with the launcher settings later and hopefully that doesn't break it, but if it does I will report back.

So strange though, my prior attempts to reinstall didn't fix the issue, but this time, I guess The Six were smiling upon me 🥲
